### PR TITLE
Debian 9 support

### PR DESCRIPTION
**Pull request by jakolehm**
Supports only cri-o because Debian does not include a docker package.

## TODO
- [x] update host-upgrades addon
- [x] use stretch as deb repo

### CODE REVIEW

File: lib/pharos/host/debian/debian.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module Pharos
+  module Host
+    class Debian < Configurer
+      def install_essentials
+        exec_script(
+          'configure-essentials.sh',
+          HTTP_PROXY: host.http_proxy.to_s,
```
Comment (kke) [inline lib/pharos/host/debian/debian.rb]: Not needed with current master

File: lib/pharos/host/debian/scripts/configure-essentials.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+
+. /usr/local/share/pharos/util.sh
+
+env_file="/etc/environment"
+
+if [ "${SET_HTTP_PROXY}" = "true" ]; then
```
Comment (kke) [inline lib/pharos/host/debian/scripts/configure-essentials.sh]: Not needed in current master

Comment (jakolehm) [reply lib/pharos/host/debian/scripts/configure-essentials.sh]: Fixed.

File: lib/pharos/host/debian/debian.rb (hunk 1 above)
Comment (jakolehm) [reply lib/pharos/host/debian/debian.rb]: Fixed.

**Comment by jakolehm** [comment]
@jnummelin @kke PTAL

**Comment by jakolehm** [comment]
```
# kubectl get nodes -o wide
NAME            STATUS    ROLES     AGE       VERSION   INTERNAL-IP     EXTERNAL-IP   OS-IMAGE                       KERNEL-VERSION   CONTAINER-RUNTIME
jari-master-0   Ready     master    44m       v1.11.3   10.133.71.215   <none>        Debian GNU/Linux 9 (stretch)   4.9.0-7-amd64    cri-o://1.11.6
jari-worker-0   Ready     <none>    44m       v1.11.3   10.133.71.198   <none>        Debian GNU/Linux 9 (stretch)   4.9.0-7-amd64    cri-o://1.11.6
jari-worker-1   Ready     <none>    44m       v1.11.3   10.133.73.88    <none>        Debian GNU/Linux 9 (stretch)   4.9.0-7-amd64    cri-o://1.11.6
jari-worker-2   Ready     <none>    44m       v1.11.3   10.133.73.40    <none>        Debian GNU/Linux 9 (stretch)   4.9.0-7-amd64    cri-o://1.11.6
```

```
# kubectl get pods --all-namespaces
NAMESPACE                NAME                                        READY     STATUS      RESTARTS   AGE
ingress-nginx            default-http-backend-787b7c796f-8gbcn       1/1       Running     0          42m
ingress-nginx            default-http-backend-787b7c796f-qlgt9       1/1       Running     0          42m
ingress-nginx            nginx-ingress-controller-hbsrk              1/1       Running     0          42m
ingress-nginx            nginx-ingress-controller-rbgrl              1/1       Running     0          42m
ingress-nginx            nginx-ingress-controller-zpthc              1/1       Running     0          42m
kontena-storage-system   kontena-storage-operator-6d98f66f8-sjzvn    1/1       Running     0          42m
kontena-storage-system   rook-ceph-agent-n9xjs                       1/1       Running     0          41m
kontena-storage-system   rook-ceph-agent-qvjql                       1/1       Running     0          41m
kontena-storage-system   rook-ceph-agent-v564s                       1/1       Running     0          41m
kontena-storage-system   rook-discover-b57rl                         1/1       Running     0          41m
kontena-storage-system   rook-discover-lxzpw                         1/1       Running     0          41m
kontena-storage-system   rook-discover-wvkpm                         1/1       Running     0          41m
kontena-storage          kontena-storage-tools-57f757bbdf-7wz6n      1/1       Running     0          42m
kontena-storage          rook-ceph-mgr-a-7979bbbc84-fhbhb            1/1       Running     0          40m
kontena-storage          rook-ceph-mon0-bn6sn                        1/1       Running     0          41m
kontena-storage          rook-ceph-mon1-szqql                        1/1       Running     0          41m
kontena-storage          rook-ceph-mon2-jw9m8                        1/1       Running     0          40m
kontena-storage          rook-ceph-osd-id-0-54966fd6c4-gzzws         1/1       Running     0          40m
kontena-storage          rook-ceph-osd-id-1-85f4768578-rgg5h         1/1       Running     0          40m
kontena-storage          rook-ceph-osd-id-2-655c6c8c89-9wms4         1/1       Running     0          40m
kontena-storage          rook-ceph-osd-prepare-jari-worker-0-dhjss   0/1       Completed   0          40m
kontena-storage          rook-ceph-osd-prepare-jari-worker-1-phl95   0/1       Completed   0          40m
kontena-storage          rook-ceph-osd-prepare-jari-worker-2-tq8p4   0/1       Completed   0          40m
kube-system              cert-manager-6884bd78fc-4jndb               1/1       Running     0          42m
kube-system              coredns-785ff588f5-7nd8p                    1/1       Running     0          45m
kube-system              coredns-785ff588f5-9ncxn                    1/1       Running     0          45m
kube-system              etcd-jari-master-0                          1/1       Running     0          44m
kube-system              host-upgrades-89kpl                         1/1       Running     0          42m
kube-system              host-upgrades-b4ccx                         1/1       Running     0          6m
kube-system              host-upgrades-s6c8g                         1/1       Running     0          42m
kube-system              host-upgrades-xqmkn                         1/1       Running     0          42m
kube-system              kube-apiserver-jari-master-0                1/1       Running     0          44m
kube-system              kube-controller-manager-jari-master-0       1/1       Running     0          44m
kube-system              kube-proxy-2sgjp                            1/1       Running     0          45m
kube-system              kube-proxy-6nbhf                            1/1       Running     0          45m
kube-system              kube-proxy-f4gqp                            1/1       Running     0          45m
kube-system              kube-proxy-p8c66                            1/1       Running     0          45m
kube-system              kube-scheduler-jari-master-0                1/1       Running     0          44m
kube-system              metrics-server-848f9fd978-gkh7m             1/1       Running     0          45m
kube-system              pharos-proxy-jari-worker-0                  1/1       Running     0          44m
kube-system              pharos-proxy-jari-worker-1                  1/1       Running     0          44m
kube-system              pharos-proxy-jari-worker-2                  1/1       Running     0          44m
kube-system              pharos-telemetry-1539612000-rcjsg           0/1       Completed   0          6m
kube-system              tiller-deploy-b8f466c7-zkdgx                1/1       Running     0          42m
kube-system              weave-net-4x5hc                             2/2       Running     0          45m
kube-system              weave-net-86mzh                             2/2       Running     1          45m
kube-system              weave-net-jr8cf                             2/2       Running     1          45m
kube-system              weave-net-pnhx5                             2/2       Running     0          45m
```

**Pull request merged**